### ISSUE TITLE
Key repeat too fast in remote control mode (resurrect issue 136)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kader990/android-xbmcremote#265]
```
XBMC Remote 0.7.0

What steps will reproduce the problem?
1. Enter remote control mode in XBMC Remote
2. Click any key -- for example up/down/left/right

What is the expected output? What do you see instead?

XBMC cursor on screen should move one position, and then begin autorepeating 
similar to the 
native XBMC IR remote.

What version of XBMC Remote are you using?

XBMC cursor on screen moves 10+ positions within a fraction of a second.

Which Android device are you using (and which ROM, if custom)?

NexusOne Froyo FRF50

Which XBMC revision are you running and on which platform?

XBOX, softmod
T3CH XBMC 2010-04-27 SVN rev29407

Please provide any additional information below.

This is basically a reopen of issue 136: http://code.google.com/p/android-
xbmcremote/issues/detail?id=136
Adding and changing XBMC settins esinitialdelay, escontinuousdelay has no 
effect: fast 
autorepeat does not improve. See details in issue 136.

Suspect this might be an issue in XBMC itself, not the remote, but could not 
find any existing 
tickets in http://trac.xbmc.org/ that mention esinitialdela or 
escontinuousdelay.

Attached is a patch to http://code.google.com/p/android-
xbmcremote/source/browse/trunk/XBMC%20Remote/src/org/xbmc/android/remote/present
atio
n/controller/RemoteController.java to send Remote keypresses with autorepeat 
disabled. This 
successfully works around the issue, but loses autorepeat. I propose to put it 
into the trunk until 
the autorepeat is sorted out, because with the fast autorepeat the remote is 
basically unusable.
```

Original issue reported on code.google.com by `klepikov@gmail.com` on 25 May 2010 at 4:48
- Merged into: #136

Attachments:
- [RemoteController.java.patch](https://storage.googleapis.com/google-code-attachments/android-xbmcremote/issue-265/comment-0/RemoteController.java.patch)
